masterlist.yaml

```
@@ -6867,3 +6867,27 @@ plugins:
     clean:
       - crc: 0x17ED1071
         util: '[FO4Edit v4.0.4](https://www.nexusmods.com/fallout4/mods/2737)'
+
+  - name: 'Another Pine Forest (Mod|Trees Only|Winter)\.esp'
+    url: [ 'https://www.nexusmods.com/fallout4/mods/54027/' ]
```

LOOT's UI makes better use of location names now, so it helps to add them:

```suggestion
    url:
      - link: 'https://www.nexusmods.com/fallout4/mods/54027/'
        name: 'Another Pine Forest Mod'
```